students/mielczarek_szymon/lesson_05_lists/the_bowling_alley.py

```diff
@@ -0,0 +1,8 @@
+num_of_pins, num_of_balls = [int(i) for i in input().split()]
+result = ['I'] * num_of_pins
+for i in range(num_of_balls):
```

Comment:
no no no, the `i` value in for iteration is not used. 
so by convention, it should be underscore `_`
please remember that!